examples/i2c/client_ds3231.c

```diff
@@ -0,0 +1,144 @@
+#include <microkit.h>
+#include <libco.h>
+#include <sddf/util/printf.h>
+#include <sddf/timer/client.h>
+#include <sddf/i2c/queue.h>
+#include <sddf/i2c/client.h>
+#include "ds3231.h"
```

Comment:
The convention is to have everything that's included from sDDF behind `<sddf/`. So in this case `<sddf/i2c/devices/ds3231.h>` instead.